### PR TITLE
US124277  - Fix up languages

### PR DESCRIPTION
**Pull request by devpow112**
This adds missing languages and also adds language verification.

### CODE REVIEW

File: lang/fr-fr.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+    "noOutcomesStudent": "Aucun(e) {outcome, select, competencies {compétence} expectations {attente} objectives {objectif} outcomes {résultat} standards {norme} other {norme}} évalué(e).",
+    "noScaleInstructor": "Pour afficher les progrès de l’apprenant par rapport à ce(tte) {outcome, select, competencies {compétence} expectations {attente} objectives {objectif} outcomes {résultat} standards {norme} other {norme}}, créez et sélectionnez une échelle de réalisation.",
+    "noScaleStudent": "Aucune échelle n’a été définie pour mesurer les réalisations par rapport aux {outcome, select, competencies {compétences} expectations {attentes} objectives {objectifs} outcomes {résultats} standards {normes} other {normes}}.",
+    "noSearchResults": "Aucun résultat trouvé pour « {searchTerm} »",
+    "notAssessed": "Non évalué",
+    "numSearchResults": "{numResults} {numResults, plural, =1 {résultat} other {résultats}} de recherche pour « {searchTerm} »",
```
Comment (devpow112) [inline lang/fr-fr.json:23]: I think the language checker doesn't like the use of  `«`? Not sure why it dislikes this line

File: build/lang/en.js [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
-		'nodeAriaTextPosition': '{position} of {count}'
-	}
-};
+import '@polymer/polymer/polymer-legacy.js';
+window.D2L = window.D2L || {};
+window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
+window.D2L.PolymerBehaviors.OutcomesUserProgress = window.D2L.PolymerBehaviors.OutcomesUserProgress || {};
+window.D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior = window.D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior || {};
+window.D2L.language = window.D2L.language || {};
+window.D2L.language.en = {
```
Comment (rgoessl) [inline build/lang/en.js:7]: Wouldn't this conflict with other components on the page?

**Review by rgoessl — CHANGES_REQUESTED** [review]

File: build/lang/en.js (hunk 2 above)
Comment (devpow112) [reply build/lang/en.js:7]: This seems to be the way we do this. It existed before I made changes in all the other language files in the `build/lang` folder and I just followed the pattern. From some local testing it didn't come up as an issue.

Comment (devpow112) [reply build/lang/en.js:7]: Oh wait. NM. You mean cause I didn't namespace `language`. Yes that may cause issue. Will fix

Comment (devpow112) [reply build/lang/en.js]: Added a namespace for the language stuff

**Review by rgoessl — APPROVED** [review]

**Pull request merged**